docs/software/uenv/build.md

```diff
@@ -0,0 +1,46 @@
+[](){#ref-uenv-build}
+# Building uenv
+
+CSCS provides a build service for uenv that takes as its input a uenv recipe, and builds the uenv using the same pipeline used to build the officially supported uenv.
+
+The command takes two arguments:
```

Comment:
Which command? =) I think it's worth mentioning it beforehand, because the sentence above is referring to the build service, and not the CLI tool/commands to interact with it.

Reply:
Fixed.